### ISSUE TITLE
The highlight can't see in other pdf reader software

### ISSUE THREAD

**Issue by Snopylz** [ahrm/sioyek#38]
I try to highlight some sentences but when I use other pdf software (like foxit reader, chrome)to open it I can't see the highlight, but if I use highlight in foxitreader, it can still be seen in sioyek. Do I miss some operation like **save the changes or pdf**?

**Comment by ahrm** [comment]
Yes, we don't store the highlight in the PDF files themselves but instead use a database file. Because we never change the PDF files, we can content-address the PDF files which means that we can easily synchronize the data across different machines. For more information see https://github.com/ahrm/sioyek/issues/29#issuecomment-915982417 .
So the fact that we don't store the highlights inside the PDFs was a conscious design decision (although I agree there are a lot of benefits to storing the highlights in the PDFs themselves too) and we probably never will support that.

**Comment by rtviii** [comment]
@ahrm would you guys be open to a contribution for like an optional '--apply_annotations' command or something that would write the highlights into the pdfs? I, as i'm sure for a lot of folks do, sync my materials across multiple machines and this is kind of the "version-control"-y step that's missing. Likewise for pulling the highlights into `shared.db` from pdfs that have them already (added elsewhere with other programs). Love the project to death.

**Comment by rtviii** [comment]
Just to clarify: i think elsewhere (like in #29) and the HN thread, people mention a number of reading devices on which it's not so easy to throw up a linux environment so syncing via just pulling our `shared.db` is not so easy. Perhaps that's niche.

**Comment by ahrm** [comment]
This is already implemented and will be available in the next version of sioyek.
